core/src/main/java/io/dekorate/utils/TopologicalSort.java

```diff
@@ -183,6 +185,11 @@ public Node(Class value) {
     public String toString() {
       return value.getName();
     }
+
+    @Override
+    public int compareTo(Node o) {
+      return value.getName().compareTo(o.value.toString());
```

Comment:
This seems awkward.

Reply:
Let's discuss this in the other comment.

Reply:
Sorry, I should be more specific. I mean that comparing `getName()` to `toString()` seems very weird.

Reply:
Very weird and certainly wrong :p 
Will fix it.
Thanks for spotting!

Reply:
Fixed